### PR TITLE
Clone legacy branch

### PR DESCRIPTION
**Pull request by johanstokking**
Resolves https://github.com/TheThingsNetwork/packet_forwarder/issues/4

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Note that this fix should be cherry-picked (or separate PR needed) for the `spi` branch

File: install.sh [hunk 1]
```diff
@@ -139,10 +141,12 @@ popd
...
 # Build packet forwarder
 if [ ! -d packet_forwarder ]; then
-    git clone https://github.com/TheThingsNetwork/packet_forwarder.git
+    git clone -b legacy https://github.com/TheThingsNetwork/packet_forwarder.git
     pushd packet_forwarder
 else
     pushd packet_forwarder
+    git fetch origin
+    git checkout legacy
     git pull
```
Comment (ralphtheninja) [inline install.sh]: `git pull` is no longer needed since we do `git fetch` combined with `git reset --hard`

**Comment by ralphtheninja** [comment]
LGTM

**Comment by gonzalocasas** [comment]
Thanks @johanstokking for the PR and thanks @ralphtheninja for reviewing it!

**Pull request merged**